plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMetastoreAccessOperations.java

```diff
@@ -206,7 +207,7 @@ public void testRefreshMaterializedView()
 
         assertMetastoreInvocations("REFRESH MATERIALIZED VIEW test_refresh_mview_view",
                 ImmutableMultiset.builder()
-                        .addCopies(GET_TABLE, 6)
+                        .addCopies(GET_TABLE, 2)
```

Comment:
That's quite an improvement. :+1: 

Maybe it is worth adding it in the description of the PR